### DISCUSSION TITLE
does sympy rely upon cloud services?

### DISCUSSION THREAD

**Discussion by harridu** [sympy/sympy#27365] [Q&A]
Hi folks, I was asked to check whether sympy relies upon any cloud services. Does it? 

Sorry for asking. A brief glimpse into the code did not reveal any cloud services, but there are tons of python modules I do not survey completely. Any helpful comment is highly appreciated.

Harri

**Comment by Iamsaikumar14** [comment]
Hi Harri,

SymPy is designed as a pure Python library for symbolic mathematics and does not inherently rely on any cloud services. Its primary focus is to perform symbolic computation locally, without needing external dependencies beyond Python modules that you install alongside it.
